src/renderer/components/appBar/appBar.scss

```diff
@@ -1,10 +1,16 @@
+@import '../../../scss/zindex';
+@import '../../../scss/variables';
+
 .appBar {
-  background-color: black;
-  border-bottom: 1px #e0e0e0 solid;
-  position: relative;
+  background: black !important;
```

Comment:
do we need to have this important tag here? Is there another way we can do this?

Reply:
I was running into some styling issues with the component framework that was used. If you can figure out how to change the color inside the framework for the app bar that would be the best practice. I think in this case it's not a huge deal.